modules/luci-mod-status/htdocs/luci-static/resources/view/status/connections.js

```diff
@@ -130,12 +131,27 @@ return view.extend({
 			var src = dns_cache[c.src] || (c.layer3 == 'ipv6' ? '[' + c.src + ']' : c.src);
 			var dst = dns_cache[c.dst] || (c.layer3 == 'ipv6' ? '[' + c.dst + ']' : c.dst);
 
+			let network = c.layer3.toUpperCase();
+			let protocol = c.layer4.toUpperCase();
+			let source ='%h'.format(c.hasOwnProperty('sport') ? (src + ':' + c.sport) : src);
+			let destination = '%h'.format(c.hasOwnProperty('dport') ? (dst + ':' + c.dport) : dst);
+			let transfer = [ c.bytes, '%1024.2mB (%d %s)'.format(c.bytes, c.packets, _('Pkts.')) ];
+
+			if  (filterText) {
+				let filterTextExpressions = filterText.split(' ');
+				if (filterTextExpressions.some((element) => element.toUpperCase !== network && element.toUpperCase() !== protocol 
```

Comment:
Should that be a function?

Reply:
In reallity it must return a 'truthy' value https://developer.mozilla.org/en-US/docs/Glossary/Truthy

You can use the complete function or like here, an arrow function: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some#examples

Reply:
You have `element.toUpperCase !== network` (not a function call)
but you have `element.toUpperCase() !== protocol` (function call)

Reply:
True, this is a bug :( I will change it. Good catch! I didn't understand your first explanation, sorry...

Reply:
Fixed!